src/renate/updaters/model_updater.py

```diff
@@ -277,34 +298,28 @@ def _load_learner(
     def _fit_learner(
         self,
         learner: Learner,
-        train_loader: DataLoader,
-        val_loader: DataLoader,
         use_syne_tune_callback: bool = True,
     ) -> None:
         callbacks: List[Callback] = []
         if use_syne_tune_callback:
-            callbacks.append(SyneTuneCallback(val_loader is not None))
+            callbacks.append(SyneTuneCallback(learner._val_enabled))
         if self._output_state_folder is not None:
             model_checkpoint_callback = RenateModelCheckpoint(
                 model=self._model,
-                input_state_folder=self._output_state_folder,
+                output_state_folder=self._output_state_folder,
                 metric=self._metric,
                 mode=self._mode,
-                val_enabled=val_loader is not None,
+                val_enabled=learner._val_enabled,
                 use_syne_tune_callback=use_syne_tune_callback,
             )
             callbacks = [model_checkpoint_callback]  # FIXME: insert at 0 as soon as PTL is fixed.
 
         if self._early_stopping_enabled:
-            if val_loader is not None:
-                callbacks.insert(
-                    0,
-                    EarlyStopping(
-                        monitor=self._metric,
-                        mode=self._mode,
-                    ),
-                )
+            if learner._val_enabled:
+                print("Using early stopping!!!!")
```

Comment:
nit: remove print and use a warning if you want to communicate something important

Reply:
Sorry, left over from debugging.